server/middlewares/stats.js

```diff
@@ -4,8 +4,11 @@ const client = redis.createClient(process.env.REDIS_URL)
 const mung = require('express-mung')
 const normalizeUrl = require('normalize-url')
 const cbor = require('borc')
+const ua = require('universal-analytics')
 
 const knownIps = new Set()
+const adaliteTrackingID = 'UA-143401606-1'
+const anonymousUserID = 555
```

Comment:
I would not declare this as a variable but rather hardcode it into "initVisitor" and leave there the comment "google analytics anonymous visitor". We are not dealing with visitors, it's just a google analytics specific, so there is no need to expose it.